src/services/dumpers/agent-nodejs.ts

```diff
@@ -205,5 +247,9 @@ export default class AgentNodeJs extends AbstractDumper {
     this.writeDockerignore();
     this.writeDockerfile();
     this.writeDockerCompose(dumpConfig);
+
+    if (schema) {
```

Comment:
```suggestion
    if (isMongoose) {
```
(I didn't test obviously, but this if(schema) -> writeMongoose seems weird to me)

Reply:
I update bot the if statement and the schema variable name.